common/benchmark/src/BenchmarkUtils.h

```diff
@@ -41,4 +40,4 @@ TB_NOINLINE static void timeLambda(L&& lambda, const std::string& message) {
 }
 
 
-#endif //TRENCHBROOM_BENCHMARKUTILS_H
+
```

Comment:
Ideally the EOF is just moved up 1 line, but I wasn't sure how to do that in VS Code find/replace, so I just replaced with an empty string.

Reply:
I think you can get this in vscode by ending the regex with `\n`, I'll see if I can adjust the PR to do this